requests/add-output.yml

```
@@ -0,0 +1,3 @@
+action: add_feedstock_output
+feedstock_to_output_mapping:
+  - cuda-ctadvisor: "cuda-ctadvisor*"
```

please be explicit about the outputs you want, globs are really discouraged.

---

We have several cases in the CUDA Toolkit we produce multiple targets and it is easier to glob them together

That said, this case has only one output. So think we can just drop the glob

```suggestion
  - cuda-ctadvisor: "cuda-ctadvisor"
```